scripts/restore

```diff
@@ -18,12 +18,8 @@ ynh_script_progression --message="Loading settings..."
 
 domain=$(ynh_app_setting_get --app $app --key domain)
 path_url=$(ynh_app_setting_get --app $app --key path)
-final_path=$(ynh_app_setting_get --app $app --key final_path)
 db_name=$(ynh_app_setting_get --app $app --key db_name)
-
-# CHECK IF THE APP CAN BE RESTORED
```

Comment:
Why do you remove this ?

Reply:
`ynh_webpath_available` is no longer needed, it is now handled by the core